src/lib/mutate-processor.ts

```diff
@@ -99,12 +108,13 @@ export async function mutateProcessor(
           errorMessage = "An error occurred with the mutate action.";
         }
 
-        Log.error(actionMetadata, `Action failed: ${errorMessage}`);
+        // Log the alias on failure
+        aliasLogger.error(actionMetadata, `Action failed: ${errorMessage}`);
         response.warnings.push(`Action failed: ${errorMessage}`);
 
         switch (config.onError) {
           case Errors.reject:
-            Log.error(actionMetadata, `Action failed: ${errorMessage}`);
+            aliasLogger.error(actionMetadata, `Action failed: ${errorMessage}`);
```

Comment:
same question around whether child logger should handle controller actions

Reply:
Updated to use Log